kart/rev_list_objects.py

```diff
@@ -96,6 +96,21 @@ def rev_list_feature_blobs(repo, start_commits, stop_commits):
     )
 
 
+TILE_POINTER_FILES_PATTERN = re.compile(r"(.+)/\.point-cloud-dataset[^/]*/tile/.+")
+
+
+def rev_list_tile_pointer_files(repo, start_commits, stop_commits):
+    """
+    Yield all the blobs with a path identifying them as features (or rows) of a "table-dataset".
```

Comment:
> Yield all the blobs with a path identifying them as features (or rows) of a **"table-dataset"**.

point cloud dataset?

Reply:
Worth adding a statement along these lines?

> Unlike git-lfs, we don't scan blob bodies to identify LFS pointer files, since we know exactly what paths will be LFS objects for a dataset.

Reply:
This is not necessarily relevant to this python file, which concerns itself only with running rev-list --objects commands, and, using them to list all the whatsits in a thingamy-dataset.

I've added a bit more documentation to the kart lfs+ push command to explain this